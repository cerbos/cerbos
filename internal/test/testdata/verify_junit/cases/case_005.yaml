---
description: Errored (due to duplicate test cases) test suite