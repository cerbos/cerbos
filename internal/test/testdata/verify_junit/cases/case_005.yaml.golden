<testsuites errors="1" failures="0" skipped="0" tests="0">
  <testsuite description="Tests for verifying the company resource policy" name="CompanyTestSuite" file="suite_test.yaml" errors="1" failures="0" skipped="0" tests="0">
    <error type="RESULT_ERRORED">Invalid test suite: another test named Company Test 1 already exists</error>
    <properties></properties>
  </testsuite>
</testsuites>